### PR TITLE
chore: add uds package CR

### PR DESCRIPTION
**Pull request by TristanHoladay**
update the sonarqube package to utilize the new operator pattern provided by uds-core

### CODE REVIEW

**Comment by TristanHoladay** [comment]
should we go ahead and implement these netpols since goal is to deploy soon on uds-prod with gitlab and runner?

```#networkPolicies:
#  enabled: true
#networkPolicy:
#  enabled: true
#  additionalNetworkPolicys:
#    ingress:
#    - from:
#      - namespaceSelector:
#          matchLabels:
#            kubernetes.io/metadata.name: gitlab-runner-sandbox
#        podSelector: {}
#      ports:
#      - port: 9000
#        protocol: TCP
#    egress:
#    - to:
#      - namespaceSelector:
#          matchLabels:
#            kubernetes.io/metadata.name: gitlab
#        podSelector:
#          matchLabels:
#            app: webservice
#    - to:
#      - namespaceSelector:
#          matchLabels:
#            kubernetes.io/metadata.name: gitlab-runner-sandbox
#        podSelector: {}
#    podSelector:
#      matchLabels: {}
#    policyTypes:
#    - Egress
#    - Ingress
```

**Comment by mjnagel** [comment]
@TristanHoladay might be worth a discussion on how we anticipate comms for pipelines working. One option is just to have all ingress go through the Istio VS, but that does mean egress on the other end is allowed to effectively anywhere. It feels "smarter" to handle traffic "directly" internal to the cluster, but we don't always know if/how runners and gitlab are deployed.  It might make more sense to leave those stitching pieces to the environment specific bundle?

**Comment by TristanHoladay** [comment]
That makes sense. So each bundle would deploy a separate package, which is a chart with another CR or traditional netpols to handle making whatever pieces exist in that environment talk to each other?

**Comment by mjnagel** [comment]
@TristanHoladay that's my leaning yeah. We have something similar to handle auth pieces for AWS currently in our environment, and I think every environment will probably have a small zarf package that does similar things. Might be good to talk through that a bit and determine if we want to publish examples/a full example package or bundle for those missing bits....

File: tasks/deploy.yaml [hunk 1]
```diff
@@ -7,4 +7,4 @@ tasks:
   - name: sonarqube-test-bundle
     actions:
       - description: Deploy the UDS bundle with Sonarqube and its dependencies
-        cmd: UDS_CONFIG=bundle/uds-config.yaml uds deploy bundle/uds-bundle-sonarqube-test-${UDS_ARCH}-*.tar.zst --confirm --no-progress
+        cmd: UDS_CONFIG=bundle/uds-config.yaml uds deploy bundle/uds-bundle-sonarqube-test-${UDS_ARCH}-*.tar.zst --confirm --no-progress -l debug
```
Comment (mjnagel) [inline tasks/deploy.yaml:10]:
```suggestion
        cmd: UDS_CONFIG=bundle/uds-config.yaml uds deploy bundle/uds-bundle-sonarqube-test-${UDS_ARCH}-*.tar.zst --confirm --no-progress
```
Not 100% sure but suspect this was a temp thing?

File: values/sonarqube-values.yaml [hunk 2]
```diff
@@ -74,9 +74,9 @@ postgresql:
...
     port: 5432
 
-service:
-  labels:
-    uds/istio-gateway: tenant
-    uds/istio-host: sonarqube
+# service:
+#   labels:
+#     uds/istio-gateway: tenant
+#     uds/istio-host: sonarqube
```
Comment (mjnagel) [inline values/sonarqube-values.yaml:81]:
```suggestion
```

**Review by mjnagel — COMMENTED** [review]
LGTM overall - 2 small comments.

**Pull request merged**